src/app/zap-templates/zcl/data-model/chip/access-control-cluster.xml

```diff
@@ -14,11 +14,17 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-<configurator>
+<!--
+XML generated by Alchemy; DO NOT EDIT.
+Source: src/data_model/ACL-Cluster.adoc
+Parameters: in-progress 
+Git: 
+-->
+<configurator xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../zcl.xsd">
   <domain name="CHIP"/>
-
   <enum name="AccessControlEntryPrivilegeEnum" type="enum8">
     <cluster code="0x001F"/>
+    <cluster code="0x0752"/>
```

Comment:
This part is not really OK: in the spec you have random other clusters "using" this cluster's types, but the spec does not allow that.   I filed https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/11471 to track this, but in the meantime, please remove these bits.

Reply:
@robszewczyk is working on a PR to stop Joint Fabric Datastore from using this enum, which will get rid of this cluster code.

Reply:
@hasty would this be an option to "errata-out in alchemy" as a stop-gap?

I wonder as we update things if we could have some robust eratta so we still have most things alchemy and as spec moves we remove erratas.

Reply:
It seems that the spec was changed to remove the dependency, regenerated file to remove the additional cluster.